docs/users/openmrs-orthanc.md

```diff
@@ -0,0 +1,36 @@
+# OpenMRS-Orthanc Flows
+
+## Flows Overview
+
+``` mermaid
+    sequenceDiagram
+        participant OpenMRS
+        participant Ozone
+        participant Orthanc
+        Orthanc->>Ozone: ImagingStudy
+        Orthanc->>Ozone: Series
+        Orthanc->>Ozone: Instance
+        Ozone->>OpenMRS: Observation (Attachment)
+```
+
+## Flows List
+
+| Source |Element| |Target|         Element          |
+|:------:|:---:|:---:|:---:|:------------------------:|
+|Orthanc |Instance|⭆|OpenMRS| Observation (Attachment) |
+
+
+!!! question "What is Observation (attachment)?"
+
+    An attachment is represented as an complex Obs in OpenMRS. An Attachment can have a file, image, pdf with an Attachment title and description, in case of Orthanc, an attachment has an image and a hyperlink to view the imaging study.
+
+## Flows Details
+
+### **1** &nbsp; Orthanc Instance → OpenMRS Observation (attachment)
+
+An Orthanc ImagingStudy has multiple series, each series has multiple instances an instance image is mapped to OpenMRS Patient Observation as an attachment.
```

Comment:
Please reformulate, something is wrong with this sentence.

Reply:
```
An Orthanc ImagingStudy consists of multiple series, each containing multiple instances. Each instance image is mapped to an OpenMRS Patient Obs as an Attachment.

```